db/seeds/run-seed.js

```diff
@@ -1,7 +1,7 @@
-const devData = require('../data/development-data/index.js');
-const seed = require('./seed.js');
-const db = require('../connection.js');
+const devData = require("../data/development-data/index.js");
+const seed = require("./seed.js");
 
+const db = require("./");
```

Comment:
Why did you change the path of your database connection file? Does this still work?